Rakefile

```diff
@@ -35,8 +35,14 @@ task :test do
           next
         end
 
+        linkText = mentor["link_text"].to_s
         url = mentor["link_url"]
-        next if url.nil? || url.empty?
+        if url.nil? || url.empty?
```

Comment:
Let's add the same `to_s` on line 39 that you did for line 38. That way we can check only against `empty?` and not against `nil`.